### PR TITLE
Update versions for 3.1

### PR DESCRIPTION
**Pull request by hardys**
Update OS/chart versions and airgap images

Aligns with recent updates to https://github.com/suse-edge/suse-edge.github.io

### CODE REVIEW

File: telco-examples/mgmt-cluster/multi-node/eib/mgmt-cluster-multinode.yaml [hunk 1]
```diff
@@ -20,39 +20,39 @@ kubernetes:
     charts:
```
[the remark this answers is not in the document]
Comment (hardys) [reply telco-examples/mgmt-cluster/multi-node/eib/mgmt-cluster-multinode.yaml:20]: Ah good catch, thanks!

File: telco-examples/mgmt-cluster/airgap/eib/mgmt-cluster-airgap.yaml [hunk 2]
```diff
@@ -89,84 +89,73 @@ kubernetes:
...
+    - name: registry.suse.com/rancher/mirrored-longhornio-livenessprobe:v2.14.0
+    - name: registry.suse.com/rancher/mirrored-longhornio-openshift-origin-oauth-proxy:4.15
+    - name: registry.suse.com/rancher/mirrored-longhornio-backing-image-manager:v1.7.1
+    - name: registry.suse.com/rancher/mirrored-longhornio-longhorn-engine:v1.7.1
+    - name: registry.suse.com/rancher/mirrored-longhornio-longhorn-instance-manager:v1.7.1
+    - name: registry.suse.com/rancher/mirrored-longhornio-longhorn-manager:v1.7.1
+    - name: registry.suse.com/rancher/mirrored-longhornio-longhorn-share-manager:v1.7.1
+    - name: registry.suse.com/rancher/mirrored-longhornio-longhorn-ui:v1.7.1
+    - name: registry.suse.com/rancher/mirrored-longhornio-support-bundle-kit:v0.0.42
+    - name: longhornio/longhorn-cli:v1.7.1
```
[the remark this answers is not in the document]
Comment (hardys) [reply telco-examples/mgmt-cluster/airgap/eib/mgmt-cluster-airgap.yaml]: Fixed, thanks!

**Review by atanasdinov — APPROVED** [review]
LGTM!

**Review by alknopfler — APPROVED** [review]
LGTM

**Pull request merged**